Neos.Flow/Classes/Package.php

```diff
@@ -71,7 +71,7 @@ public function boot(Core\Bootstrap $bootstrap)
                 }
             }
         });
-        $dispatcher->connect(Cli\SlaveRequestHandler::class, 'dispatchedCommandLineSlaveRequest', Persistence\PersistenceManagerInterface::class, 'persistAll');
+        $dispatcher->connect(Cli\SlaveRequestHandler::class, 'dispatchedCommandLineSlaveRequest', Persistence\PersistenceManagerInterface::class, 'persistAll', false);
```

Comment:
```suggestion
        $dispatcher->connect(Cli\SlaveRequestHandler::class, 'dispatchedCommandLineSlaveRequest', function () use ($bootstrap) {
            $persistenceManager = $bootstrap->getObjectManager()->get(Persistence\PersistenceManagerInterface::class);
            $persistenceManager->persistAll(true);
        });
```
Alternative that keeps the `persistAll(trueish)` semantic from before. Not sure that is wanted?

Reply:
I think I am fine with this for now

Reply:
No, I'm pretty sure this never was intended. And since a slave request does not have any notion of "safe requests" the expectation is to have things persist, IMHO.